native-activity/app/src/main/cpp/main.cpp

```diff
@@ -319,18 +225,12 @@ static void engine_handle_cmd(android_app* app, int32_t cmd) {
  * event loop for receiving input events and doing other things.
  */
 void android_main(android_app* state) {
-  Engine engine{};
+  Engine engine;
```

Comment:
neither {} nor memset()?

(did you mean to do `Engine engine = { .app = state };` but stop part way through?)

Reply:
Reworked this to be an actual constructor. The behavior was the same whether `{}` was present before or not (both invoked the default constructor), but a real constructor is clearer.